common/Fabric-Install-Utilities.psm1

```diff
@@ -277,123 +277,170 @@ function Get-AccessToken($authUrl, $clientId, $scope, $secret)
     return $accessTokenResponse.access_token
 }
 
-function Add-ApiRegistration($authUrl, $body, $accessToken)
+function Save-ApiRegistration($authUrl, $body, $accessToken)
```

Comment:
Renaming this function is a breaking change for others who might be using it, Perhaps we keep the name the same?